CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
+means we will never make a backwards-incompatible change within a major version of the project.
+
+## v3.0.0
```

Comment:
Ah, I see what you mean. I agree that we should go the `praw` route and set this at 0.1.0 or something.